### ISSUE TITLE
No resizing in new versions of linux

### ISSUE THREAD

**Issue by MintDja** [ssokolow/quicktile#121]
At the moment, at home and at work, I use the LinuxMint 19.3 system on which everything works successfully. At the weekend I decided to try LinuhMint 20.1 XFCE at home after debian 10.
Both on one and on the other system, the filmed behavior is observed. Windows move around the hotkey, but do not resize. Regards
https://youtu.be/wGqtHBpTUak

**Comment by ssokolow** [comment]
It's probably because you changed `ModMask` to include `<Shift>` but didn't delete or change these lines from your config file:

```
<Shift>KP_1 = move-to-bottom-left
<Shift>KP_2 = move-to-bottom
<Shift>KP_3 = move-to-bottom-right
<Shift>KP_4 = move-to-left
<Shift>KP_5 = move-to-center
<Shift>KP_6 = move-to-right
<Shift>KP_7 = move-to-top-left
<Shift>KP_8 = move-to-top
<Shift>KP_9 = move-to-top-right
```

I don't have time to double-check right now, but those are probably taking precedence.

**Comment by MintDja** [comment]
Yes, I changed ModMask = <Ctrl><Alt> to ModMask = <Ctrl><Shift>
Goodnight

**Comment by ssokolow** [comment]
Does removing the `<Shift>KP_*` lines fix the problem?

...because it looks like it's operating as designed to me. (You specified that `<Ctrl><Shift><Shift>KP_1` should be `move-to-bottom-left`, which is the same as `<Ctrl><Shift>KP_1`, and that's what it's doing.)

**Comment by MintDja** [comment]
yes, the extra lines are to blame! Thank you!

**Comment by MintDja** [comment]
in this case there are extra lines in the downloaded configuration file

**Comment by ssokolow** [comment]
Yes. The default configuration is to have holding down `<Shift>` mean "Don't resize, just move."

It's your job to remove the `<Shift>` lines if you add `<Shift>` to `ModMask`.

...though, as I said, now that you've made me aware of that point of confusion, I'll try to remember to add a warning to the next version.

**Comment by ssokolow** [comment]
Actually, let's keep this open and rename it as my reminder to add that warning.

**Comment by ssokolow** [comment]
It's looking like the solution for this will be the new config GUI I'm working on, where the hotkey editor will not have a `ModMask` (each binding will be exactly what you pressed to set it) and will highlight any conflicting keybindings in red.
